### PR TITLE
CB-16569 adding the missing jsonignore property to CredentialResponse

### PR DESCRIPTION
**Pull request by doktoric**
See detailed description in the commit message.

### CODE REVIEW

**Review by bergerdenes — APPROVED** [review]

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/ImageCatalogV4Endpoint.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ ImageV4Response getImageFromDefault(@PathParam("workspaceId") Long workspaceId,
             @PathParam("govCloud") boolean govCloud) throws Exception;
 
     @GET
-    @Path("image/type/{type}/provider/{provider}")
+    @Path("image/type/{type}/provider/{provider}/govCloud/{govCloud}")
```
Comment (lajosrodek) [inline core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/ImageCatalogV4Endpoint.java:183]: Sanity check: will this have any effects on other clients calling this endpoint, e.g. CDP CLI?

Comment (doktoric) [reply core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/ImageCatalogV4Endpoint.java]: I just removed this. Seems no client using it. This endpoint will be not govcloud compatible

**Comment by lnardai** [comment]
@doktoric  Shouldn't this be targeted to 2.54?

**Comment by doktoric** [comment]
@lnardai yes because the deadline for this is March 28th

**Comment by doktoric** [comment]
govcloud is not presented on on 2.54

**Review by bbihari — APPROVED** [review]

**Pull request merged**